src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -6,7 +6,7 @@
 import * as fse from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
-import { gitignoreFileName, hostFileName, localSettingsFileName, ProjectRuntime, proxiesFileName } from '../../../constants';
+import { gitignoreFileName, hostFileName, localSettingsFileName, ProjectLanguage, ProjectRuntime, proxiesFileName } from '../../../constants';
```

Comment:
If you follow my comment below, I don't think you need this anymore.

Reply:
We still need ProjectLanguage since Managed Dependencies is only supported in PowerShell.